### ISSUE TITLE
Animation does not work without item id.

### ISSUE THREAD

**Issue by dotneet** [storybookjs/react-treebeard#224]
As he mentioned in the comment below, animations does not work without item id.
https://github.com/storybookjs/react-treebeard/issues/223#issuecomment-507031668

**Comment by dotneet** [comment]
@maximilianoforlenza 
Thank you for maintaining very useful package.
I found out that this change breaks animation.

https://github.com/storybookjs/react-treebeard/commit/f8eec1ed4078a0e94cfb3c824675f9a251319dba

Why use randomString() instead of index?

**Comment by maximilianoforlenza** [comment]
Because index as a key is an anti-pattern

**Comment by dotneet** [comment]
That's correct. But using randomString() in rendering time does not resolve any problem that index as a key cause.
Because keys are changed every rendering time.

**Comment by FilipHarald** [comment]
Is there any work around for this? Currently I'm adding `.id` to my nodes in the collection so that the animation is working. But I would prefer to use their `._id` or have `randomString()` work with animations.

@maximilianoforlenza , thanks for an awesome package!
